app/lib/account_deletion_config.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module AccountDeletionConfig
+  module_function
+
+  def config
+    @config ||= load_config
+  end
+
+  def valid?
+    @valid ||= check_validity
```

Comment:
Do not memoize boolean. If you do that, it will always call `check_validity` in case it returns`false`

Reply:
Right, but not memorizing means it will call it always :( I can also call it only if the instance variable is not defined... Wdyt @hallelujah?

Reply:
Yes you can use that